extensions/parquet/table/src/test/resources/e0/.gitattributes

```diff
@@ -0,0 +1 @@
+*.parquet filter=lfs diff=lfs merge=lfs -text
```

Comment:
Should we be using binary instead of text?

Reply:
This command is automatically generated on running `git lfs track "*.parquet"`

The `-text` part specifies that Git should treat these files as binary files rather than text files. This is important to avoid any end-of-line conversion issues.
So we do want to keep it like this.

Reference: 
https://earthly.dev/blog/a-developer-guide-to-git-lfs/
https://stackoverflow.com/a/70075078